components/brave_rewards/resources/ui/components/pageWallet.tsx

```diff
@@ -67,6 +69,37 @@ class PageWallet extends React.Component<Props, State> {
     })
   }
 
+  onModalBackupOnCopy = (backupKey: string) => {
+    const success = clipboardCopy(backupKey)
+    // TODO(jsadler) possibly flash a message that copy was completed
+    console.log(success ? 'Copy successful' : 'Copy failed')
```

Comment:
@jasonrsadler can you please create follow up issue for this one. I agree that we need to show confirmation to the user that copy was successful